### PR TITLE
A bit entity brain/anger stuff

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

**Review by Juuxel — APPROVED** [review]

File: mappings/net/minecraft/entity/ai/brain/task/WalkTowardClosetAdultTask.mapping [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_5355 net/minecraft/entity/ai/brain/task/WalkTowardClosetAdultTask
```
Comment (YanisBft) [inline mappings/net/minecraft/entity/ai/brain/task/WalkTowardClosetAdultTask.mapping]:
```suggestion
CLASS net/minecraft/class_5355 net/minecraft/entity/ai/brain/task/WalkTowardClosestAdultTask
```
file too

File: mappings/net/minecraft/entity/ai/brain/task/TakeJobSiteTask.mapping [hunk 2]
```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_5327 net/minecraft/entity/ai/brain/task/TakeJobSiteTask
+	FIELD field_25158 speed F
+	METHOD <init> (F)V
+		ARG 1 speed
+	METHOD method_29260 canUseJobSite (Lnet/minecraft/class_4158;Lnet/minecraft/class_1646;Lnet/minecraft/class_2338;)Z
+		ARG 1 poiType
```
Comment (YanisBft) [inline mappings/net/minecraft/entity/ai/brain/task/TakeJobSiteTask.mapping:6]: I think we used `poi` in other classes

Comment (liach) [reply mappings/net/minecraft/entity/ai/brain/task/TakeJobSiteTask.mapping:6]: We do for `PointOfInterestType`? I think we use poi for poi instances, do we?

Comment (i509VCB) [reply mappings/net/minecraft/entity/ai/brain/task/TakeJobSiteTask.mapping:6]: Yeah `PointOfInterestType` is like `StatusEffect` as `PointOfInterest` is to `StatusEffectInstance`

**Pull request merged**